e2e/tsconfig.json

```diff
@@ -1,22 +1,8 @@
 {
+  "extends": "../tsconfig.json",
```

Comment:
Let’s `extend` this one too? Why not?

Hm.. Just noticed that `"noEmit": true` went missing. Shall I bring it back? And perhaps shall I use it for all `__tests__`?